Frontend/ui-driver/src/Screens/CancellationRateScreen/View.purs

```diff
@@ -0,0 +1,364 @@
+      ]
+      [ linearLayout
+        [ height WRAP_CONTENT
+        , width MATCH_PARENT
+        , orientation VERTICAL
+        , margin $ MarginHorizontal 16 16
+        ]
+        [ cancellationRateGauge state push
+        , cancellationRateData state push
+        , cancellationRateReduceTrivia state push configs
+        , cancellationRateDownsidesTrivia state push configs
+        ]
+      ]
+    ]
+  where
+    getLanguage lang = 
+      let language = DS.toLower $ DS.take 2 lang
+      in if not (DS.null language) then "_" <> language else "_en"
+
+headerLayout :: forall w. ST.CancellationRateScreenState -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w
+headerLayout state push =
+  linearLayout
+    [ width MATCH_PARENT
+    , height WRAP_CONTENT
+    , orientation VERTICAL
+    ]
+    [ linearLayout
+        [ width MATCH_PARENT
+        , height MATCH_PARENT
+        , orientation HORIZONTAL
+        , layoutGravity "center_vertical"
+        , padding (Padding 5 12 5 12)
+        ]
+        [ imageView
+            [ width $ V 40
+            , height $ V 40
+            , imageWithFallback $ fetchImage FF_ASSET "ny_ic_back"
+            , gravity CENTER_VERTICAL
+            , onClick push $ const BackPressed
+            , padding (Padding 10 10 10 10)
+            , margin (MarginLeft 5)
+            , cornerRadius 20.0
+            , rippleColor Color.rippleShade
+            ]
+        , textView
+            $ [ width WRAP_CONTENT
+              , height MATCH_PARENT
+              , text (getString RIDE_CANCELLATION_RATE)
+              , margin (MarginLeft 10)
+              , color Color.black
+              , weight 1.0
+              , gravity CENTER_VERTICAL
+              ]
+            <> FontStyle.h3 TypoGraphy
+        ]
+    , horizontalLine 0 0
+    ]
+
+cancellationRateGauge :: forall w. ST.CancellationRateScreenState -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w
+cancellationRateGauge state push =
+   linearLayout
+    [ height WRAP_CONTENT
+    , width MATCH_PARENT
+    , orientation VERTICAL
+    , background Color.aliceBlueLight
+    , cornerRadius 24.0
+    , margin $ Margin 0 20 0 0
+    , padding $ Padding 16 16 16 16
+    ]
+    [ linearLayout
+      [ height WRAP_CONTENT
+      , width MATCH_PARENT
+      , orientation HORIZONTAL
+      ] 
+      [ 
+        relativeLayout
+        [ height $ V 103
+        , width $ V $ ((screenWidth unit)/ 2) - 32
+        , gravity CENTER
+        ][ 
+          imageView 
+          [ imageWithFallback $ fetchImage FF_ASSET "ny_ic_gauge_image"
+          , width $ V $ ((screenWidth unit)/ 2) - 32
+          , height $ V 100
+          , gravity LEFT
+          ]
+        , linearLayout
+          [ gravity CENTER
+          , width MATCH_PARENT
+          ][
+            imageView
+            [ imageWithFallback $ fetchImage FF_ASSET "ny_ic_gauge_tip"
+            , width $ V $ round (((toNumber (screenWidth unit - 32)/ 2.0) ) * (11.0 / 75.0))
```

Comment:
What is the logic of 11 / 75 here?

Reply:
Aesthetically pleasing, Sui width should be 11/75 of the gauge width